modes/passenger-services.md

```diff
@@ -0,0 +1,185 @@
+# Mobility Data Specification: **Passenger Servies**
+
+Passenger Services refers to taxis and transportation network companies (TNCs).  Passenger Services typically have a driver, one or more passengers, and multiple passengers may be on different trips.  The state machine tracks the trip states of the passengers separately from the vehicle state.  
+
+See the [modes overview](/modes) for how the mode specific information below applies across MDS.
+
+## Taxi vs. TNC implementation differences
+
+Taxis typically require explicit tracking of maintenance.
+TNCs typically do not.
+
+TODO `provider_cancellation` (more common in TNC) vs `driver_cancellation` (more common in taxi)
+
+## Table of Contents
+
+* [Mode Attributes](#mode-attributes)
+* [Vehicle States](#vehicle-states)
+* [Event Types](#event-types)
+* [Vehicle State Events](#vehicle-states-events)
+* [State Machine Diagram](#state-machine-diagram)
+
+## Mode Attributes
+
+### Mode ID
+
+TODO `passenger-services`
+
+### Journey ID
+
+The `journey_id` field shall have a consistent value in overlapping trips, e.g. "pooled" or "shared" rides with different start and/or end locations.
+
+### Trip ID Requirements
+
+Events require a valid `trip_id` in events where `event_types` contains `reservation_start`, `reservation_stop`, `trip_start`, `trip_stop`, `trip_end`, `passenger_cancellation`, or `driver_cancellation`. 
+
+Additionally, `trip_id` is required if `event_types` contains a `enter_jurisdiction` or `leave_jurisdiction` event pertaining to a passenger trip. 
+
+### Trip Type
+
+The `trip_type` field is not currently used in passenger services.
+
+### Trip Attributes
+
+The `trip_attributes` array is not (yet? TODO) used with in passenger services.
+
+### Vehicle Attributes
+
+The `vehicle_attributes` array may have the following key value pairs:
+
+- `year` (integer)
+- `make` (string)
+- `model` (string)
+- TODO others?
+
+### Propulsion Types
+
+#### Valid for vehicle_types: car
+
+| `propulsion`      | Description                         |
+| ----------------- | ----------------------------------- |
+| `electric`        | Contains a battery-powered motor    |
+| `combustion`      | Contains a gas engine-powered motor |
+
+### Accessibility Options
+
+This enum represents the accessibility options available on a given vehicle, or the accessibility options utilized for a given trip. This enum is currently only used by the Taxi mode, and is not used by micromobility.
```

Comment:
Note to change taxi to passenger services.

Reply:
Fixed in da59502